bin/update-patterns

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+set -e
+
+function finish {
+    docker rm browser-pattern-library &> /dev/null
```

Comment:
the container may not be existing on exit (if the script errored out sooner than expected), `|| true` may help to make clear this is not the source of the error